paddle/fluid/framework/ir/alloc_continuous_space_for_grad_pass.cc

```diff
@@ -23,15 +23,16 @@
 #include "paddle/fluid/framework/ir/graph_helper.h"
 #include "paddle/fluid/framework/op_registry.h"
 
-DEFINE_uint64(fuse_parameter_memory_size, 0,  // 0 KB
+DEFINE_uint64(fuse_parameter_memory_size, 0,  // Bytes
```

Comment:
I think that maybe KB is better.

Reply:
KB is not enough now. We use MB commonly.
But MB is more coarse-grained.